core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -1146,6 +1151,46 @@ private void parseMetrics(ConfigurationReader reader, ConfigurationBuilderHolder
       ParseUtils.requireNoContent(reader);
    }
 
+   private void parseTracing(ConfigurationReader reader, ConfigurationBuilderHolder holder) {
+      GlobalConfigurationBuilder builder = holder.getGlobalConfigurationBuilder();
+      for (int i = 0; i < reader.getAttributeCount(); i++) {
+         ParseUtils.requireNoNamespaceAttribute(reader, i);
+         String value = reader.getAttributeValue(i);
+         Attribute attribute = Attribute.forName(reader.getAttributeName(i));
+         switch (attribute) {
+            case COLLECTOR_ENDPOINT: {
+               builder.tracing().collectorEndpoint(value);
+               break;
+            }
+            case ENABLED: {
+               boolean enabled = ParseUtils.parseBoolean(reader, i, value);
+               if (enabled) {
+                  builder.tracing().enabled();
```

Comment:
`enabled()` does not enable anything. I'm guessing testing is a little bit lacking. 
```suggestion
                  builder.tracing().enable();
```

Reply:
Applied thank you